x/plans/types/geolocation.go

```diff
@@ -65,15 +62,26 @@ func ParseGeoEnum(arg string) (geoloc int32, err error) {
 }
 
 func GetAllGeolocations() []int32 {
-	return allGeoEnumRegionsList
+	var allGeos []int32
+	for i := 0; i < geoAmount; i++ {
+		allGeos = append(allGeos, 1<<i)
```

Comment:
lets do this in the init?

Reply:
done. see https://github.com/lavanet/lava/commit/69fd139f8c489583fca76490a1f2b5aee4978a59